### DISCUSSION TITLE
View object in dropdown (question)

### DISCUSSION THREAD

**Discussion by berkantsahn** [primefaces/community#686] [PrimeNG]
`Hi everyone,`

I'm trying to add a datatable into dropdown but while doing this I noticed that the components were not working properly...

Is it possible to create such a structure with primeng?
What I'm talking about here is not just displaying the data grid in the dropdown. I'm talking about being able to view any other object inside an object.

Thank you in advance for your help.

**Comment by cetincakiroglu** [comment]
Hi,

Could you please share a stackblitz example so we can identify the issue clearly?

**Comment by cetincakiroglu** [comment]
Closing the topic due to the inactivity.
